src/Apps/Order/Routes/NewPayment/index.tsx

```diff
@@ -0,0 +1,496 @@
+    }
+  }
+
+  onContinue: () => void = () => {
+    this.setState({ isCommittingMutation: true }, () => {
+      if (this.needsAddress()) {
+        const { errors, hasErrors } = validateAddress(this.state.address)
+        if (hasErrors) {
+          this.setState({
+            isCommittingMutation: false,
+            addressErrors: errors,
+            addressTouched: this.touchedAddress,
+          })
+          return
+        }
+      }
+
+      const { address } = this.state
+      const stripeBillingAddress = this.getStripeBillingAddress(address)
+      this.props.stripe
+        .createToken(stripeBillingAddress)
+        .then(({ error, token }) => {
+          if (error) {
+            this.setState({
+              isCommittingMutation: false,
+              stripeError: error,
+            })
+          } else {
+            this.createCreditCard({ token: token.id, oneTimeUse: true })
+          }
+        })
+        .catch(e => {
+          this.onMutationError(new ErrorWithMetadata(e))
+        })
+    })
+  }
+
+  @track((props, state, args) => {
+    const showBillingAddress = !args[0]
+    if (showBillingAddress) {
+      return {
+        action_type: Schema.ActionType.Click,
+        subject: Schema.Subject.BNMOUseShippingAddress,
+        flow: "buy now",
+        type: "checkbox",
+      }
+    }
+  })
+  handleChangeHideBillingAddress(hideBillingAddress: boolean) {
+    if (!hideBillingAddress) {
+      this.setState({
+        address: {
+          ...emptyAddress,
+          country: "US",
+        },
+      })
+    }
+
+    this.setState({ hideBillingAddress })
+  }
+
+  onAddressChange: AddressChangeHandler = (address, key) => {
+    const { errors } = validateAddress(address)
+    this.setState({
+      address,
+      addressErrors: {
+        ...this.state.addressErrors,
+        [key]: errors[key],
+      },
+      addressTouched: {
+        ...this.state.addressTouched,
+        [key]: true,
+      },
+    })
+  }
+
+  render() {
+    const { order } = this.props
+    const {
+      stripeError,
+      isCommittingMutation,
+      address,
+      addressErrors,
+      addressTouched,
+    } = this.state
+
+    return (
+      <>
+        <HorizontalPadding px={[0, 4]}>
+          <Row>
+            <Col>
+              <OrderStepper currentStep="Payment" steps={["Payment"]} />
+            </Col>
+          </Row>
+        </HorizontalPadding>
+        <HorizontalPadding>
+          <TwoColumnLayout
+            Content={
+              <Flex
+                flexDirection="column"
+                style={isCommittingMutation ? { pointerEvents: "none" } : {}}
+              >
+                {order.mode === "OFFER" && (
+                  <>
+                    <Flex>
+                      <CountdownTimer
+                        action="Respond"
+                        note="Expiration will end negotiations on this offer. Keep in mind the work can be sold to another buyer in the meantime."
```

Comment:
In zeplin this copy is just "Expired offers end the negotiation process". I think the one you have here is better but we should check with Stephen.

Another question for Stephen, should the action be "Respond by"? Seems like an inappropriate way to describe updating one's payment information. Maybe "Submit by"? Nice and general.

Reply:
Hey @ds300 - Good question. I believe this countdown messaging applies to the Seller and Buyer, and not all scenarios involve submitting a payment (someone could reject the offer/counter offer as a response), which I imagine that was Brian's reasoning. I think for this particular scenario with 'Add a new payment' page, "Submit by" seems more appropriate since this is specifically geared towards payment submission. Is that a quick text change?